Quantum-Control-Applications/NV-Centers/Confocal-Setup/configuration.py

```diff
@@ -75,10 +74,10 @@ def IQ_imbalance(g, phi):
     },
     "elements": {
         "NV": {
-            "mixInputs": {"I": ("con1", 1), "Q": ("con1", 2), "lo_frequency": NV_LO_freq, "mixer": "mixer_qubit"},
+            "mixInputs": {"I": ("con1", 1), "Q": ("con1", 2), "lo_frequency": NV_LO_freq, "mixer": "mixer_NV"},
             "intermediate_frequency": NV_IF_freq,
             "operations": {
-                "const": "cw_pulse",
+                "cw": "const_pulse",
```

Comment:
Just to understand and also modify the flux tunable transmon files if needed, why do you want to use "cw" instead of "const"?

Reply:
Yes, I wanted to make it consistent, it was different between the NV and the SC.
I randomly decided that the operation will be "cw".